llvm/test/Analysis/CostModel/RISCV/rvv-intrinsics.ll

```diff
@@ -1501,6 +1501,113 @@ define void @vp_fadd(){
 }
 
 
+define void @vp_fneg() {
```

Comment:
It would be better to merge test cases into `arith-fp.ll` which already contains cost model tests for `fneg`.

Reply:
done

Reply:
Tests are still in `rvv-intrinsics.ll`?

I thought moving these test cases into `CostModel/RISCV/arith-fp.ll` which already contains some tests for non-vp `fneg` would better.

Reply:
fixed

Reply:
If there is no problem, can I merge it. Byw,pls review #114516